arelle/plugin/validate/EDINET/rules/upload.py

```diff
@@ -670,11 +670,10 @@ def rule_uri_references(
                 modelObject=uriReference.element,
             )
             continue
-        fullPath = Path(uriReference.document.uri).parent / path
-        if not val.modelXbrl.fileSource.exists(str(fullPath)):
+        if not path.suffix:
```

Comment:
I'm not sure this is quite right. A file doesn't have to include a suffix. Does is_dir() with a zipfile.Path work here?

Reply:
In the context of an EDINET filing, a file without a suffix would fire suffix validation errors, but I'll check out your suggestion.